CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.9.6
+
+
+### New Features
+
+- SSI: Introduces generic safeguards for automatic instrumentation when using single step install in the form of early exit conditions. Early exit from instrumentation is triggered if a version of software in the environment is not explicitly supported by ddtrace. The Python runtime itself and many Python packages are checked for explicit support on the basis of their version.
+
```

Comment:
Given that we merged this to unblock merging [the one pipeline change](https://github.com/DataDog/dd-trace-py/pull/10529) (which removes these changes), I think we can remove this note.
```suggestion
```